Dockerfile

```diff
@@ -0,0 +1,9 @@
+# renovate: datasource=docker depName=containerbase/buildpack
+ARG SIDECAR_VERSION=2.2.2
```

Comment:
Not BUILDPACK_VERSION ?

Reply:
No, as it's used for local image name. 😉

But we can change this, then i need a config option in `builder.json` to point to the right name.